src/utils/hooks/useCustomForm.ts

```diff
@@ -1,9 +1,9 @@
 import { FieldValues, useForm, UseFormProps, UseFormReturn } from "react-hook-form"
-import { yupResolver } from "@hookform/resolvers/yup"
-import type { AnyObjectSchema } from "yup"
+import { zodResolver } from "@hookform/resolvers/zod"
+import type { ZodSchema } from "zod"
 
 interface HookProps {
-  schema: AnyObjectSchema
+  schema: ZodSchema
```

Comment:
It is better to use `T extends z.ZodTypeAny` constraint here.
https://github.com/colinhacks/zod#writing-generic-functions

Reply:
Done